### PR TITLE
initial spring boot configuration

### PR DESCRIPTION
**Pull request by ponyataeva**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>core</groupId>
```
Comment (ManWithNoNamePL) [inline core/pom.xml]: can we make it com.tweet instead ?

File: core/pom.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <artifactId>core</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.0.5.RELEASE</version>
+        <relativePath />
+    </parent>
+
```
Comment (ManWithNoNamePL) [inline core/pom.xml]: lets add:
	<properties>
		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
		<java.version>1.9</java.version>
	</properties>

File: core/pom.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-actuator</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+        </dependency>
+    </dependencies>
+
```
Comment (ManWithNoNamePL) [inline core/pom.xml]: <build>
		<plugins>
			<plugin>
				<groupId>org.springframework.boot</groupId>
				<artifactId>spring-boot-maven-plugin</artifactId>
			</plugin>
		</plugins>
	</build>

File: core/pom.xml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>core</groupId>
+    <artifactId>core</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
```
Comment (ManWithNoNamePL) [inline core/pom.xml]: packaging jar - make jar not war :)

**Comment by ponyataeva** [comment]
updated request, please have a look it again

**Comment by ManWithNoNamePL** [comment]
initial app

**Pull request merged**